competency-explorer/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
```

Comment:
You mentioned the ejecting as being because you wanted access to the Markdown files outside this package, right? A couple of thoughts:
* You could move the markdown files _into_ the package
* You could add a setup script that creates symlinks for all the markdown files

Thoughts? 😁

Reply:
I thought about it, but github pages (without an associated circle workflow) won't recompile the bundle. If I imported directly through webpack, every competency change would also need to re-run the webpack build, whereas with the current setup, competency changes will be dynamically loaded without any changes needed to the app.

Reply:
Oh, I may have misunderstood your comment. For moving the files, I'd like to avoid it so that the default repo view is still the competencies. I also avoided symlinks because I've been burned by them not being retained, but it looks like GH may be able to preserve the symlink; I don't know how that would affect the GH pages view, but I can try it out.

In any case, everything I found on `create-react-app` said that without ejecting, you couldn't fetch or import markdown files, so I don't think either approach would have prevented ejecting.